datafusion/optimizer/src/simplify_expressions/expr_simplifier.rs

```diff
@@ -1530,14 +1531,26 @@ mod tests {
 
         // Check non string arguments
         // to_timestamp("2020-09-08T12:00:00+00:00") --> timestamp(1599566400i64)
-        let expr =
-            call_fn("to_timestamp", vec![lit("2020-09-08T12:00:00+00:00")]).unwrap();
-        test_evaluate(expr, lit_timestamp_nano(1599566400000000000i64));
+        //
+        // todo - determine how to migrate this
```

Comment:
I currently don't know how this could be enabled again because of the issues with the fact that this calls into private code in this module. Any ideas would be appreciated.

Reply:
I think the const evaluator is called via the simplfier

Thus I think we could move these tests to the `datafusion/core/tests/simplification.rs` (and call the entire simplifier on them, rather than just the constant propagator)  as well as they are really tests that ensure simplification works with these particular functions.

Reply:
Done.